libraries/AC_PID/AC_HELI_PID.cpp

```diff
@@ -23,17 +23,17 @@ const AP_Param::GroupInfo AC_HELI_PID::var_info[] = {
     // @Param: VFF
     // @DisplayName: Velocity FF FeedForward Gain
     // @Description: Velocity FF Gain which produces an output value that is proportional to the demanded input
-    AP_GROUPINFO("VFF",    4, AC_HELI_PID, _ff, 0),
+    AP_GROUPINFO("VFF",    4, AC_HELI_PID, _kff, 0),
```

Comment:
should we perhaps name the parameter variable "_vff" instead of "_kff" to make it match the parameter name?

Reply:
addressed with parameter ordering